common/supplemental/supplementalData.xml

```diff
@@ -1307,7 +1307,7 @@ XXX Code for transations where no currency is involved
 		<language type="amo" scripts="Latn"/>
 		<language type="an" scripts="Latn"/>
 		<language type="ang" scripts="Latn" alt="secondary"/>
-		<language type="ann" scripts="Latn" territories="NG"/>
+		<language type="ann" scripts="Latn" territories="NG"/> <!-- re-added manually in CLDR-15922 -->
 		<language type="anp" scripts="Deva"/>
 		<language type="aoz" scripts="Latn"/>
 		<language type="ar" scripts="Arab" territories="AE BH DJ DZ EG EH ER IL IQ JO KM KW LB LY MA MR OM PS QA SA SD SO SY TD TN YE"/>
```

Comment:
Shouldn't there have been an automatically-generated change for ckb in here that does *not* need to be manually reverted? (see my comments in https://unicode-org.atlassian.net/browse/CLDR-14992?focusedCommentId=166413)

Reply:
@pedberg-icu I've now re-instated the change.